helm/hpcc/values.schema.json

```diff
@@ -473,6 +473,19 @@
     "resources": {
       "type": "object"
     },
+    "memory": {
```

Comment:
The possible settings should be documented in the jira, together with how they are processed.  (It was very hard to review the code without it.)